src/components/screens/ServiceTile.tsx

```diff
@@ -253,10 +253,11 @@ const stylesWithProps = ({
   safeAreaInsets: EdgeInsets;
 }) => {
   const containerPadding = normalize(14);
+  const tilePadding = normalize(7);
```

Comment:
Is `tilePadding` always the half of `containerPadding`? Then we could have `containerPadding / 2` here or avoid the new const at all and do `width: tileSize - (containerPadding / 2)` at the end?